package.json

```diff
@@ -60,6 +54,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@types/bluebird": "*",
```

Comment:
I don't think we're using bluebird in the API, any idea why a dependency for its types is included in here?

Reply:
Because of another dependency's badly designed typescript side of things. They require types/bluebird but don't specify it in their package.json and in turns results in a not found error thrown by TS